src/layer/AtmosphereLayer.js

```diff
@@ -0,0 +1,414 @@
+/*
+ * Copyright (C) 2014 United States Government as represented by the Administrator of the
+ * National Aeronautics and Space Administration. All Rights Reserved.
+ */
+/**
+ * @exports AtmosphereLayer
+ */
+define([
+        '../error/ArgumentError',
+        '../shaders/GroundProgram',
+        '../layer/Layer',
+        '../util/Logger',
+        '../geom/Matrix',
+        '../geom/Matrix3',
+        '../geom/Sector',
+        '../shaders/SkyProgram',
+        '../geom/Vec3'
+    ],
+    function (ArgumentError,
+              GroundProgram,
+              Layer,
+              Logger,
+              Matrix,
+              Matrix3,
+              Sector,
+              SkyProgram,
+              Vec3) {
+        "use strict";
+
+        /**
+         * Constructs a layer showing the Earth's atmosphere.
+         * @alias AtmosphereLayer
+         * @constructor
+         * @classdesc Provides a layer showing the Earth's atmosphere.
+         * @param {URL} nightImageSource optional url for the night texture.
+         * @augments Layer
+         */
+        var AtmosphereLayer = function (nightImageSource) {
+
+            Layer.call(this, "Atmosphere");
+
+            //image for the night texture
+            this._nightImageSource = nightImageSource || '../images/dnb_land_ocean_ice_2012.png';
```

Comment:
The base path for the default image must be WorldWind.configuration.baseUrl + "images/dnb_land_ocean_ice_2012.png"

See BMNGOneImageLayer for an example.